Readme.md

```diff
@@ -0,0 +1,73 @@
+# SellEverywhere
+
+## Overview
+We want to create a hardware shop using popular technologies in a friendly team
+(MKdir). Examples are any store selling goods. Current idea is to implement an
+admin panel and user panel. The basic interface of the store was designed in [Figma](https://www.figma.com/file/Re4tXW7aNCWOzed7wI1GKe/dirStore).
+
+### Basic flow
+![Basic flow](./docs/basic_flow.png)
+
+### The application must have the following functionality from the administrator perspective:
+
+* authorization - jwt;
```

Comment:
do you plan to make implementation with separate refresh \ auth tokens

Reply:
https://gist.github.com/zmts/802dc9c3510d79fd40f9dc38a12bccfc

Reply:
https://t.me/why_jwt_is_bad

Reply:
Updated readme

Reply:
I still don't see mention of 2 tokens used (or 1 token)

Reply:
found it later in text, ok